app/_layout.tsx

```diff
@@ -48,6 +49,7 @@ function Wrapper({ children }: PropsWithChildren) {
   return (
     <ThemeProvider value={colorScheme === "dark" ? DarkTheme : DefaultTheme}>
       <GluestackUIProvider config={extendedThemeConfig}>
+        <ToastEventHandler />
```

Comment:
I'm wondering if we shouldn't call it `ToastCreator`. What do think about?

Reply:
Why not ? It's more understandable